packages/react/src/themes/teams/components/Checkbox/checkboxVariables.ts

```diff
@@ -1,22 +1,75 @@
-import { CheckboxVariables as BaseCheckboxVariables } from '../../../base/components/Checkbox/checkboxVariables'
+import { pxToRem } from '../../../../lib'
 
-export type CheckboxVariables = Partial<BaseCheckboxVariables>
+export interface CheckboxVariables {
+  rootPadding: string
+
+  checkboxTextColor: string
+  checkboxBackground: string
+  checkboxBorderColor: string
+  checkboxBorderStyle: string
+  checkboxBorderRadius: string
+  checkboxBorderWidth: string
+  checkboxIndicatorColor: string
+  checkboxGap: string
+  checkboxMargin: string
+  checkboxPadding: string
+
+  checkboxTextColorHover: string
+  checkboxBorderColorHover: string
+  checkboxCheckedBackgroundHover: string
+  checkboxCheckedTextColor: string
+  checkboxCheckedBackground: string
+  checkboxCheckedBorderColor: string
+  checkboxCheckedIndicatorColor: string
+
+  toggleBorderRadius: string
+  toggleIndicatorColor: string
+  toggleMargin: string
+  togglePadding: string
+  toggleCheckedPadding: string
+
+  disabledColor: string
+  disabledCheckboxBackground: string
+  disabledCheckboxBackgroundChecked: string
+  disabledCheckboxBorderColor: string
+  disabledCheckboxCheckedIndicatorColor: string
+  disabledToggleIndicatorColor: string
+}
```

Comment:
Is there any reason why we can't use variables from `base` theme? If we need something custom we can define them:

```tsx
export type CheckboxVariables = Partial<BaseCheckboxVariables> & {
  additionalVariableForTeamsTheme1: string
  additionalVariableForTeamsTheme2: string
}
```

Reply:
I thought teams theme was supposed to stand alone from base theme - and also changes to base theme could impact teams unexpectedly?

Reply:
Base theme is intended to be the base theme for all themes.  It should only include values and variables that are applicable to all themes.  Things like "buttons have horizontal and vertical padding".  Base theme variables should be exposed for these things, the lowest common denominators for all themes.